test/table/table_padding.py

```diff
@@ -0,0 +1,533 @@
+from math import sin
+from pathlib import Path
+
+import qrcode, pytest
+
+from fpdf import FPDF
+from fpdf.enums import MethodReturnValue, YPos, TableCellFillMode, VAlign
+from fpdf.fonts import FontFace
+from fpdf.table import Padding
+from test.conftest import assert_pdf_equal, LOREM_IPSUM
+
+
+HERE = Path(__file__).resolve().parent
+
+
+def run_comparison(pdf, name, tmp_path):
+    filename = HERE / f"{name}.pdf"
+
+    try:
+        assert_pdf_equal(pdf, filename, tmp_path, generate=False)
+    except AssertionError as e:
+        # windows only debugging
+        import subprocess
+
+        # subprocess.Popen(rf'explorer "{str(filename)}"')
+        pdf.output(rf"c:\temp\{name}.pdf")
+        subprocess.Popen(rf'explorer "c:\temp\{name}.pdf"')
+
+        raise e
```

Comment:
As mentioned in my previous review, this debug code is A) specific to Windows and B) won't work in the GitHub Actions pipelines, and hence should be removed from this PR

Reply:
Hi Lucas, can you respond to my earlier question about what to do with the reference pdfs ?: https://github.com/PyFPDF/fpdf2/pull/797#issuecomment-1645260993

Until that is solved I need to do visual inspections of the generated pdfs.